website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+
+- [React Native Frameworks](/blog/2024/04/15/0.74-post#react-native-frameworks)
+- New layout behaviors
+- Support for `align-content: 'space-evenly'`
+- New Architecture support for `position: 'static'`
+- [Preparing for the New Architecture?](/blog/2024/04/15/0.74-post#preparing-for-the-new-architecture)
+- [Batched `onLayout` updates (New Architecture)](/blog/2024/04/15/0.74-post#batched-onlayout-updates-new-architecture)
+- [Removal of deprecated `PropTypes`](/blog/2024/04/15/0.74-post#removal-of-deprecated-proptypes)
+- [Deprecated APIs in `PushNotificationIOS`](/blog/2024/04/15/0.74-post#deprecated-apis-in-pushnotificationios)
+- [Android SDK minimum bump (Android 6.0)](/blog/2024/04/15/0.74-post#android-sdk-minimum-bump-android-60)
+- [Yarn 3 is the default for new projects](/blog/2024/04/15/0.74-post#yarn-3-for-new-projects)
+
+<!--truncate-->
+
+## React Native Frameworks
+
+- https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0759-react-native-frameworks.md
+  - Contextualize what changes are coming into 0.74 from proposal
+- Warning on using react-native init https://github.com/facebook/react-native/commit/a6397afb7f4279e70afa025fe6f9fa0b8e3de3fd
+
+## New layout behaviors
+
+React Native 0.74 includes [changes to Yoga](https://yogalayout.dev/blog/announcing-yoga-3.0) which enable more correct layout. These changes make styling more predictable, and enable rendering components written for the web with greater fidelity.
+
+React Native intentionally preserves some incorrect layout behaviors, where fixing them was found to effect a signigicant number of real-world components. Layout conformance will be able to be configured more granularly in future versions of React Native.
+
+:::warning
+
+React Native [previously flipped](https://yogalayout.dev/blog/announcing-yoga-3.0#correct-handling-of-logical-edges-in-row-reverse-containers) `left`/`right` (and `start`/`end`) edges when dealing with `margin`, `padding`, or `border`, set on a `row-reverse` container. Code which previously relied on edges being inverted may need to be updated to continue rendering correctly.
+
+<table>
+
+<tr>
+<td>Style</td><td>Before</td><td>After</td>
+</tr>
+
+<tr>
+
+<td width="350">
+
+```jsx
+<View
+  style={{
+    flexDirection: 'row',
+    backgroundColor: 'red',
+    margin: 10,
+    width: 200,
+    height: 100,
+  }}>
+  <View
+    style={{
+      flexDirection: 'row-reverse',
+      backgroundColor: 'blue',
+      flex: 1,
+      marginLeft: 50,
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        height: '50%',
+        flex: 1,
+        marginLeft: 50,
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+
+<td>
+
+![Previous layout](../static/blog/assets/0.74-row-reverse-before.png)
+
+</td>
+
+<td>
+
+![New layout](../static/blog/assets/0.74-row-reverse-after.png)
+
+</td>
+
+</tr>
+
+</table>
+
+:::
+
+## Support for `align-content: 'space-evenly'`
```

Comment:
Could this be a sub-bullet of `## New layout behaviors`? It seems like a new situational feature, and it has the same relevance of the above one, which is a bigger change. Hierarchy organization can help users understand better the impact of the changes we are pushing, right?